packages/app-builder-lib/templates/nsis/include/extractAppPackage.nsh

```diff
@@ -90,5 +90,45 @@
 !macroend
 
 !macro extractUsing7za FILE
-    Nsis7z::Extract "${FILE}"
+  Push $OUTDIR
+  CreateDirectory "$PLUGINSDIR\7z-out"
```

Comment:
Note: NSIS promises that `$PLUGINSDIR` is a temporary folder that is removed after the installation.